CanvasColorSpaceProposal.md

```diff
@@ -152,7 +150,15 @@ This is in contrast with luminance, which is to be clamped to the maximum standa
 
 ### ImageBitmap
 
-ImageBitmap objects (unless created with ``colorSpaceConversion="none"``) should keep track of their internal color space, and should store their contents at highest fidelity possible, subject to implementation limitations.
+IDL:
+<pre>
+partial dictionary ImageBitmapOptions {
+  CanvasColorSpaceEnum colorSpace = "srgb";
+}
+</pre>
+
+When creating an ``ImageBitmap``, if the ``colorSpaceConversion`` entry of the specified ``ImageBitmapOptions`` is not ``"none"``, then the internal color space for the resulting ``ImageBitmap`` should be the color space that is specified by the ``colorSpace`` entry of the ``ImageBitmapOptions``.
+If that ``ImageBitmap`` is then used as input to populate a WebGL or WebGPU texture, then the pixel values written to the texture should represent the ``ImageBitmap``'s source contents in the specified color space.
```

Comment:
What happens when the `colorSpace` of the image bitmap does not match the `colorSpace` of the drawing context at a time of texture upload?

Reply:
Note that even if we didn't have the new UNPACK_COLORSPACE_CONVERSION_WEBGL behavior, we'd still have this problem. The default behavior (on some implementations) is to convert to sRGB, and that conversion would need to disabled somehow.

I'll clarify that "Beware that the ImageBitmap may be converted to the WebGLRenderingContext color space during texture upload unless UNPACK_COLORSPACE_CONVERSION_WEBGL is set to NONE". I'll add that to the PR in the morning.

There is an alternative, which is to say that ImageBitmaps are special and cause UNPACK_COLORSPACE_CONVERSION_WEBGL to always be treated as NONE. That feels like something that will become messy to me.

Note for another change to add: There is no such thing as a "none" color space (such a thing breaks any color management system). Internally, "none" is "replace the profile with sRGB without converting". This should be updated to indicate that "none" is "replace the profile with `colorSpace` without converting".

Reply:
WebGL spec defines `UNPACK_COLORSPACE_CONVERSION_WEBGL` only for `HTMLImageElement`. It should not affect image bitmaps.

Reply:
In WebGL when ImageBitmaps are uploaded to textures, *no* colorspace conversion, y-flip, etc. are done. This is by design. The `UNPACK_COLORSPACE_CONVERSION_WEBGL`, `UNPACK_FLIP_Y_WEBGL` and `UNPACK_PREMULTIPLY_ALPHA_WEBGL` context settings are all ignored when uploading ImageBitmaps; these settings are applied during `ImageBitmap` creation.

The same rule will apply for the new `colorSpace` setting. The context's setting will be ignored when upoading an `ImageBitmap`.

Reply:
Thanks for the clarification. I've updated the language to make this (lack of) interaction clear.